api/src/pcapi/routes/backoffice_v3/templates/accounts/get.html

```diff
@@ -1,9 +1,6 @@
 {% extends "layouts/accounts.html" %}
-
 {% block accounts_main_content %}
-
-    {% include "accounts/get/general.html" %}
-    {% include "accounts/get/credit.html" %}
-    {% include "accounts/get/details.html" %}
-
-{% endblock %}
\ No newline at end of file
+  {% include "accounts/get/general.html" %}
+  {% include "accounts/get/credit.html" %}
+  {% include "accounts/get/details.html" %}
+{% endblock accounts_main_content %}
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** Est-ce qu'il y a besoin de rajouter `accounts_main_content` sur la fin du bloc ? ‏

<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏`{% endblock %}` tout simplement

Reply:
ça lève une erreur, de façon général, la syntax open + close tag est fragile, j'ai retrouvé pas mal d'erreur déjà dans notre code.

C'est pourquoi je préfère `jade` à html, car l'erreur avec un seul tag ouvrant est moins risqué.

On a un template avec 2 blocks, je pense qu'il vaut mieux  laisser la règle pour éviter les ambiguités.